tests/reporter/kubernetes.go

```diff
@@ -899,7 +899,8 @@ func (r *KubernetesReporter) logLogs(virtCli kubecli.KubevirtClient, logsdir str
 	}
 
 	for _, pod := range pods.Items {
-		for _, container := range pod.Spec.Containers {
+		allContainers := append(pod.Spec.Containers, pod.Spec.InitContainers...)
```

Comment:
do we really need all containers? Aren't the containers+sidecar containers enough?

Reply:
the logs you're going to need for proper debugging could be in any of these containers